tests/pytests/test_short_read.py

```diff
@@ -27,7 +27,7 @@
     'short-reads/redisearch_2.2.0_rejson_2.0.0.rdb.zip',
     'short-reads/redisearch_2.8.0.rdb.zip',
     'short-reads/redisearch_2.8.4.rdb.zip',
-    'short-reads/redisearch_2.8.12_rejson_2.0.0.rdb.zip',
+    'short-reads/redisearch_2.10.3.rdb.zip',
```

Comment:
Why did you remove the 2.8.12 rdb test? is it not relevant?

Reply:
The new rdb contains both empty TAG and TEXT, and the name was a bit misleading since we did not merge this into 2.8.